File: semestre-1/algoritmos/MR-Alg-03/MR-Alg-03-Ex-03.py
alfabeto = "abcdefghijklmnopqrstuvwxyz"

letra = input("Digite uma letra: ").lower()
if len(letra) != 1:
    print("Digite exatamente 1 caracter!")
elif letra not in alfabeto:
    print("DIgite uma letra válida!")
elif letra in "aeiou":
    print(f"{letra} é uma vogal")
else:
    print(f"{letra} é uma consoante")

File: semestre-1/algoritmos/MR-Alg-03/.c/MR-Alg-03-Ex-03.c
/*Escreva um programa Python que peça para o usuário uma letra do
alfabeto. Se o usuário entrar com as letras a, e, i, o ou u, o programa deve exibir uma
mensagem dizendo que a letra é uma vogal. Caso contrário, o programa deve exibir a
mensagem informando que a letra é uma consoante.*/

#include <stdio.h>
#include <ctype.h>

void main() {
    char letra;
    printf("Digite uma letra: ");
    scanf("%c", &letra);

    switch (tolower(letra)) {
        case 'a':
        case 'e':
        case 'i':
        case 'o':
        case 'u':
            printf("Vogal");
            break;
        default:
            printf("Consoante");
            break;
    }
}
